### PR TITLE
QA fix 1

### PR DESCRIPTION
**Pull request by DhaaraniCIT**

### CODE REVIEW

File: scripts/setup_env.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ const environment = {
   qbd_api_url: `${process.env.QBD_API_URL ? process.env.QBD_API_URL : '{{QBD_API_URL}}'}`,
   si_api_url: `${process.env.SI_API_URL ? process.env.SI_API_URL : '{{SI_API_URL}}'}`,
   sage300_api_url: `${process.env.SAGE300_API_URL ? process.env.SAGE300_API_URL : '{{SAGE300_API_URL}}'}`,
+  netsuite_api_url: `${process.env.NETSUITE_API_URL ? process.env.NETSUITE_API_URL : '{{NETSUITE_API_URL}}'}`,
```
Comment (ashwin1111) [inline scripts/setup_env.js:10]: pls add to integrations central if not already done

File: src/app/branding/branding-config.ts [hunk 2]
```diff
@@ -169,9 +169,9 @@ const kbArticles: KbArticle = {
                 SKIP_EXPORT: 'https://help.fylehq.com/en/articles/7882821-how-to-skip-exporting-specific-expenses-from-fyle-to-sage-intacct'
             },
             TRAVELPERK: {
-                PAYMENT_PROFILE_SETTINGS: 'https://help.fylehq.com/en/articles/7882821-how-to-skip-exporting-specific-expenses-from-fyle-to-sage-intacct',
-                ADVANCED_SETTING: 'https://help.fylehq.com/en/articles/7882821-how-to-skip-exporting-specific-expenses-from-fyle-to-sage-intacct',
-                LANDING: 'https://help.fylehq.com/en/articles/7882821-how-to-skip-exporting-specific-expenses-from-fyle-to-sage-intacct'
+                PAYMENT_PROFILE_SETTINGS: 'https://help.fylehq.com/en/articles/7193187-how-to-set-up-the-fyle-travelperk-integration#h_0f8ebdfa10',
+                ADVANCED_SETTING: 'https://help.fylehq.com/en/articles/7193187-how-to-set-up-the-fyle-travelperk-integration#h_281acb3026',
+                LANDING: 'https://help.fylehq.com/en/articles/7193187-how-to-set-up-the-fyle-travelperk-integration'
             }
```
Comment (ashwin1111) [inline src/app/branding/branding-config.ts:175]: update these urls for co key as well

File: src/app/shared/components/configuration/configuration-select-field/configuration-select-field.component.html [hunk 3]
```diff
@@ -7,7 +7,7 @@
 
             <div>
                 <h5 class="tw-text-text-secondary tw-text-14-px !tw-font-500">{{ label | snakeCaseToSpaceCase}}<app-mandatory-field *ngIf="isFieldMandatory"></app-mandatory-field></h5>
-                <h5 *ngIf="subLabel || (formControllerName !== 'reimbursableExportType' && formControllerName!=='cccExportType' && formControllerName !== 'creditCardExportType')" class="!tw-text-text-muted tw-text-14-px tw-pt-8-px !tw-font-400 !tw-leading-4 tw-max-w-600-px tw-text-pretty" [innerHTML]="subLabel"></h5>
+                <h5 *ngIf="subLabel || (formControllerName !== 'reimbursableExportType' && formControllerName!=='cccExportType' && formControllerName !== 'creditCardExportType')" class="!tw-text-text-muted tw-text-14-px tw-pt-8-px !tw-font-400 !tw-leading-4 tw-max-w-600-px tw-text-pretty" >{{subLabel}}<span *ngIf="exportConfigurationIconPath && appName === AppName.TRAVELPERK" class="tw-pl-4-px">Click<a (click)="showExportTable()" class="tw-cursor-pointer tw-text-link-primary"> here</a> for Preview</span></h5>
```
Comment (ashwin1111) [inline src/app/shared/components/configuration/configuration-select-field/configuration-select-field.component.html:10]: we send <b> tags in subLabel, can you place it as innerHTML? cab add ng if to existing h5 tag no?

File: src/app/shared/components/configuration/configuration-select-field/configuration-select-field.component.scss [hunk 4]
```diff
@@ -59,5 +59,5 @@
 }
 
 :host ::ng-deep .p-disabled {
-    @apply tw-text-select-disabled-text tw-bg-select-disabled-bg tw-border-select-disabled-border tw-border-solid tw-opacity-100 #{!important};
+    @apply tw-text-text-secondary tw-bg-select-disabled-bg tw-border-select-disabled-border tw-border-solid tw-opacity-100 #{!important};
```
Comment (ashwin1111) [inline src/app/shared/components/configuration/configuration-select-field/configuration-select-field.component.scss:62]: this should ideally be tw-text-select-disabled-text, let's discuss in tomo's call

File: src/app/shared/components/configuration/configuration-step-footer/configuration-step-footer.component.html [hunk 5]
```diff
@@ -1,4 +1,4 @@
-<div class="tw-flex tw-pt-24-px tw-border-t-border-tertiary" [ngClass]="!showBackButton ? 'tw-justify-end': 'tw-justify-between'">
```
Comment (ashwin1111) [inline src/app/shared/components/configuration/configuration-step-footer/configuration-step-footer.component.html:1]: this affects other apps, this shouldn't be removed

File: src/app/shared/components/configuration/preview-dialog/preview-dialog.component.html [hunk 6]
```diff
@@ -1,6 +1,11 @@
 <div>
     <p-dialog [header]="header" [dismissableMask]="true" [showHeader]="true" [modal]="true" [(visible)]="isPreviewDialogVisible" (onHide)="close()" [breakpoints]="{ '960px': '75vw' }" [draggable]="false" [resizable]="false">
-        <div class="tw-flex tw-items-center tw-justify-center tw-px-24-px tw-pb-24-px">
+        <ng-template pTemplate="header">
+            <div class="">
+               <span class="tw-text-20-px tw-font-500 tw-text-modal-header-text-color">{{header}}</span> 
+            </div>
+        </ng-template>
+        <div class="tw-flex tw-items-center tw-justify-center tw-mt-24-px tw-px-24-px tw-pb-24-px">
```
Comment (ashwin1111) [inline src/app/shared/components/configuration/preview-dialog/preview-dialog.component.html:8]: please test other apps which uses this component

File: src/app/shared/components/helper/app-landing-page-header/app-landing-page-header.component.html [hunk 7]
```diff
@@ -8,14 +8,16 @@
                     <img src="{{ iconPath }}" width="logoWidth" height="44px" [ngClass]="logoStyleClasses" />
                 </div>
                 <div class="tw-pl-16-px tw-flex tw-flex-col">
-                    <div class="tw-flex tw-pt-6-px">
+                    <div class="tw-flex tw-items-center tw-pt-6-px">
```
Comment (ashwin1111) [inline src/app/shared/components/helper/app-landing-page-header/app-landing-page-header.component.html:11]: please test all apps for this component change

File: src/assets/scss/tab-secondary.scss [hunk 8]
```diff
@@ -1,9 +1,9 @@
 :host ::ng-deep .p-tabmenu {
-    @apply tw-text-text-primary tw-text-14-px tw-font-nav-link-font-weight;
```
Comment (ashwin1111) [inline src/assets/scss/tab-secondary.scss:2]: why is this removed?

File: src/assets/scss/tab-secondary.scss [hunk 9]
```diff
@@ -1,9 +1,9 @@
...
+    @apply tw-text-14-px tw-font-nav-link-font-weight;
 }
 
 :host ::ng-deep .p-tabmenu-nav .p-menuitem-text {
-    @apply tw-text-text-primary tw-font-nav-link-font-weight;
```
Comment (ashwin1111) [inline src/assets/scss/tab-secondary.scss:6]: why is this removed?

File: src/styles.scss [hunk 10]
```diff
@@ -49,10 +49,12 @@ p {
 
 .pi-chevron-down.fyle {
     content: url('./assets/icons/arrow-down.svg') !important;
+    @apply tw-ml-12-px #{!important} ;
```
Comment (ashwin1111) [inline src/styles.scss:52]: let's not add margin here, might break other pages

File: src/styles.scss [hunk 11]
```diff
@@ -49,10 +49,12 @@ p {
...
 }
 
 .pi-chevron-down.co {
     content: url('./assets/icons/co/grv-chevron-down.svg') !important;
+    @apply tw-ml-12-px #{!important} ;
```
Comment (ashwin1111) [inline src/styles.scss:57]: let's not add margin here, might break other pages

File: src/assets/scss/tab-secondary.scss (hunk 9 above)
Comment (DhaaraniCIT) [reply src/assets/scss/tab-secondary.scss:6]: add a new variable

File: src/app/integrations/travelperk/travelperk-shared/travelperk-advanced-settings/travelperk-advanced-settings.component.ts [hunk 12]
```diff
@@ -150,9 +150,9 @@ export class TravelperkAdvancedSettingsComponent implements OnInit {
   private setupPage(): void {
     this.isOnboarding = this.router.url.includes('onboarding');
     forkJoin([
-      this.travelperkService.getTravelperkAdvancedSettings().pipe(catchError(() => of(null))),
-      this.travelperkService.getCategories()
-    ]).subscribe(([travelperkAdvancedSettingsResponse, travelperkDestinationAttribute]) => {
+      this.travelperkService.getTravelperkAdvancedSettings().pipe(catchError(() => of(null)))
+      // This.travelperkService.getCategories()
```
Comment (ashwin1111) [inline src/app/integrations/travelperk/travelperk-shared/travelperk-advanced-settings/travelperk-advanced-settings.component.ts:154]: remove if not used

File: src/app/shared/components/configuration/configuration-import-field/configuration-import-field.component.html [hunk 13]
```diff
@@ -86,11 +86,13 @@ <h5 *ngIf="!isCloneSettingView" class="!tw-text-text-muted tw-text-14-px !tw-fon
                                 {{ option.display_name }}
                             </div>
                             </ng-template>
-                            <ng-template pTemplate="selectedItem" let-item class="tw-flex tw-justify-between tw-items-center">
```
Comment (ashwin1111) [inline src/app/shared/components/configuration/configuration-import-field/configuration-import-field.component.html:89]: revert this file changes and check if this is still happening, I've fixed it in other commit and it's merged to master already. If that is working fine, this is not required

File: src/app/shared/components/input/search/search.component.ts [hunk 14]
```diff
@@ -47,6 +49,12 @@ export class SearchComponent implements OnInit {
     if (!this.form.value[this.formControllerName]) {
       this.isSearchFocused = false;
     }
+    this.searchFocused.emit(false);
```
Comment (ashwin1111) [inline src/app/shared/components/input/search/search.component.ts:52]: do it inside if block

**Review by ashwin1111 — APPROVED** [review]
